src/Attachment.php

```diff
@@ -197,4 +197,58 @@ public function mediaStrings(array $strings): array
 
         return $strings;
     }
+
+    /**
+     * @see wp_make_content_images_responsive
+     */
+    public function makeContentImagesResponsive(string $content): string
+    {
+        if (!preg_match_all('/<img [^>]+>/', $content, $matches)) {
+            return $content;
+        }
+
+        $selectedImages = $attachmentIds = [];
+
+        foreach ($matches[0] as $image) {
+            $hasSrcset = strpos($image, ' srcset=') !== false;
+            $hasClassId = preg_match('/wp-image-([0-9]+)/i', $image, $classId);
+            $attachmentId = !$hasSrcset && $hasClassId ? absint($classId[1]) : null;
+            if ($attachmentId) {
+                /*
+                * If exactly the same image tag is used more than once, overwrite it.
+                * All identical tags will be replaced later with 'str_replace()'.
+                */
+                $selectedImages[$image] = $attachmentId;
+                // Overwrite the ID when the same image is included more than once.
+                $attachmentIds[$attachmentId] = true;
+            }
+        }
+
+        if (count($attachmentIds) > 1) {
+            /*
+            * Warm the object cache with post and meta information for all found
+            * images to avoid making individual database calls.
+            */
+            _prime_post_caches(array_keys($attachmentIds), false, true);
```

Comment:
In this private method, why do you leave here a function name with underline, without CamelCase? @oxyc

Reply:
It's a WP function. Basically the entire function is a copy of WP's [`wp_make_content_images_responsive`](
https://developer.wordpress.org/reference/functions/wp_make_content_images_responsive/) as I couldn't see a way to fix srcset support without replacing it entirely.

Reply:
Ah, new for me. Thx.